mobify/platform-scaffold
# Web 1439: Cleanup the empty cart's layout

Cleanup the empty cart's layout

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1439

## Changes
- add padding top to empty cart
- make error color consistent and match the SDK 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- check empty cart and form errors
